File: C++/palindrom.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;

int main()
{
	int i;
	cin>>i;
	
	int q = i;
	int reminder = 0;
	int result =0;
	
	while(q != 0)
	{
		reminder = q%10;
		result = result*10 + reminder;
		q = q / 10;
	}
	
	if(result == i)
	{
		cout<<"Yes";
	}
	else
	{
		cout<<"No";
	}
}

File: Dart/palindrom.dart
import 'dart:io';

main() {
  int n = int.parse(stdin.readLineSync()!);

  int result = 0, reminder = 0, q = n;

  while (q != 0) {
    reminder = q % 10;
    result = (result * 10) + reminder;
    q = (q / 10).toInt();
  }

  if (result == n) {
    print("Yes");
  } else {
    print("No");
  }
}
